docs/monitoring/08-02-create-and-configure-grafana-dashboard.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Create a Grafana dashboard
+type: Tutorials
+---
+
+This tutorial shows how to create and configure a basic Grafana dashboard of a [Gauge](https://grafana.com/docs/features/panels/singlestat/#gauge) type. The dashboard shows how the values of the `cpu_temperature_celsius` metric change in time, representing the current processor temperature ranging from `60` to `90` Celsius degrees. The dashboard shows explicitly when the CPU temperature exceeds the pre-defined threshold of `75` Celsius degrees.
+
+## Prerequisites
+
+This tutorial is a follow-up of the [**Observe application metrics**](#tutorials-observe-application-metrics) tutorial that uses the `monitoring-custom-metrics` example. This example deploys the `sample-metrics-8081` service which exposes the `cpu_temperature_celsius` metric. That configuration is required to complete this tutorial.
+
+## Steps
+
+Follow these sections to create the Gauge dashboard type for the `cpu_temperature_celsius` metric.
+
+### Create the dashboard
+
+1. Navigate to Grafana. It is available under the `https://grafana.{DOMAIN}` address, where `{DOMAIN}` is the domain of your Kyma cluster, such as `https://grafana.34.63.57.190.xip.io` or `https://grafana.nightly.a.build.kyma-project.io/`. To access it from the Console UI, click **Stats & Metrics** on the left navigation menu.
```

Comment:
I'm not sure if we should mention nightly cluster in examples as it is not available for everyone.

Reply:
As discussed, I ill change it into `http://kyma.example.com/`.